contracts/MyNFT.sol

```diff
@@ -57,9 +64,14 @@ contract MyNFT is ERC721URIStorage, Ownable {
         return newItemId;
     }
 
-    function doTheRace() public view returns (uint256) {
-        uint256 index = random() % _tokenIds.current();
-        return index;
+    function doTheRace() public returns (uint256) {
+        lastRaceResult = random() % _tokenIds.current();
+        console.log("lastRaceResult: ", lastRaceResult);
+        return lastRaceResult;
```

Comment:
strangely enough this does not return integer in js, but some object.

Reply:
it is also modifying the blockchain now, so have to pay...

Reply:
? what object

Reply:
idk, very strange.. obj with some hash and other fields.